packages/appkit/tests/appkit.test.ts

```diff
@@ -896,10 +905,12 @@ describe('Base', () => {
       vi.spyOn(AccountController, 'state', 'get').mockReturnValue(mockAccountData as any)
 
       appKit = new AppKit({ ...mockOptions })
+      const adapter = { getBalance: vi.fn().mockResolvedValue({ balance: '0.00', symbol: 'ETH' }) }
```

Comment:
`getBalance` already mocked in `tests/mocks/Adapter.ts`, see `balance.test.ts` for examples